src/styles/content.scss

```diff
@@ -228,6 +228,33 @@ table.os-raise-horizontaltable {
   }
 }
 
+table.os-raise-textheavyadjustedtable {
```

Comment:
Would this be equivalent to something like the following?:

```scss
%os-raise-textheavytable-base {
  @extend %os-raise-basictable;

  width: 100%;

  th,
  td {
    text-align: left;
  }

  th {
    vertical-align: bottom;
  }

  td {
    vertical-align: top;
  }

  td:first-child {
    border-left: none;
  }
}

table.os-raise-textheavytable {
  @extend %os-raise-textheavytable-base;

  table-layout: fixed;
}

table.os-raise-textheavyadjustedtable {
  @extend %os-raise-textheavytable-base;

  table-layout: auto;
}
```

If so, would that be easier to maintain / read?

Reply:
Yes it would - thanks for the suggestion.